### PR TITLE
feat(GeoSearch): expose useGeoSearch() hook in RISH

### PR DESCRIPTION
**Pull request by VladislavFitz**
**Summary**

Adds `useGeoSearch()` hook to Reach InstantSearch Hooks

**Result**

### CODE REVIEW

File: examples/react-hooks/getting-started/src/index.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import './env';
```
Comment (Haroenv) [inline examples/react-hooks/getting-started/src/index.tsx:1]: let's undo this change in this PR

````suggestion
````

**Review by Haroenv — COMMENTED** [review]
connector lgtm

**Review by Haroenv — APPROVED** [review]

**Review by dhayab — APPROVED** [review]
Looks good!
